### PR TITLE
A smol change

### PR DESCRIPTION
**Pull request by norbcodes**
PLEASE MERGE THIS

### CODE REVIEW

**Review by mf366-dev — APPROVED** [review]
Everything seems fine with `.gitignore`.

**Comment by norbcodes** [comment]
Yeah but i don't wanna commit my venv and .idea folder lol

File: minigames/PongEngine.py [hunk 1]
```diff
@@ -34,7 +34,7 @@ def LoadText():
         NUMBERS.append(temp)
 
 def PrintText(text):
-    sys.stdout.write(text)  # lel
+    print(text, end="")  # lel
```
[the remark this answers is not in the document]
Comment (norbcodes) [reply minigames/PongEngine.py:37]: I ran the game through PyCharm and it was laggy, so I changed this. But then I realized that it runs fine through actual CMD so this change is really unnecessary.

**Review by mf366-dev — APPROVED** [review]
Everything ready to be merged, branch won't be deleted in order for @norbcodes to reuse it if he needs to.

File: minigames/PongEngine.py (hunk 1 above)
[the remark this answers is not in the document]
Comment (MF366-Coding) [reply minigames/PongEngine.py:37]: Alright, thank you!

**Comment by norbcodes** [comment]
Thank you @MF366-Coding

**Pull request merged**